packages/preset-inline-svg/index.js

```diff
@@ -0,0 +1,37 @@
+const webpackFinal = (webpackConfig = {}, options = {}) => {
+  const { module = {} } = webpackConfig;
+  const { svgLoaderOptions } = options;
+  return {
+    ...webpackConfig,
+    module: {
+      ...module,
+      rules: [
+        ...(module.rules || []).map(rule => {
+          const ruleTestString = rule.test.toString();
+          if (/svg\|?/.test(ruleTestString)) {
+            return {
+              ...rule,
+              test: new RegExp(
+                ruleTestString
+                  .replace(/svg\|?/, '') // Remove other svg rules
+                  .replace(/\//, ''), // No "/" needed when creating a new RegExp
```

Comment:
This could use some scrutiny. There have been some posts, [such as this](https://github.com/storybookjs/storybook/issues/7360#issuecomment-538799239), which suggest a different strategy for overriding the rule, however I wanted to be a bit more broad in my replacement of the rule just in case Storybook updates it with more file types. Maybe there is another approach that is even better than this?

Reply:
Mind making this into a function & writing a few test cases for this?

Reply:
No problem. I'll update when I get some free time!